extra/end-to-end-test.rb

```diff
@@ -0,0 +1,66 @@
+require 'rest-client'
+require 'json'
+require 'pp'
+
+# subnets and domains must exist on Infoblox
+URL = "http://localhost:8448"
+DNS_DOMAIN = "ibx.lan"
+DNS_IPV4 = "192.168.202.13"
+DNS_IPV6 = "2001:db8::1"
+DNS_PTR4 = "13.202.168.192.in-addr.arpa"
+DNS_PTR6 = "1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.8.b.d.0.1.0.0.2.ip6.arpa"
+
+def do_get(path)
+  RestClient.get("#{URL}#{path}")
+end
+
+def do_post(path, body = nil, params = {})
+  RestClient.post("#{URL}#{path}", body, params.merge({content_type: :json}))
+end
+
+def do_delete(path, params = {})
+  RestClient.delete("#{URL}#{path}", params.merge({content_type: :json}))
+end
+
+features = JSON.parse(do_get("/v2/features"))
+
+if features["dns"]["state"] == "running"
+  fqdn = "end2end.#{DNS_DOMAIN}"
+  cname = "alias-end2end.#{DNS_DOMAIN}"
+  puts "Performing DNS tests with #{fqdn}"
+  begin
+    do_post("/dns?type=a&fqdn=#{fqdn}&value=#{DNS_IPV4}")
```

Comment:
It's very weird to me that it even accepts these as GET parameters. IMHO the proxy shouldn't accept that.

Reply:
Yeah, I haven't noticed but it worked. It's a different story tho.